teamC/dayoung/controllers/postController.js

```diff
@@ -0,0 +1,95 @@
+const postService = require('../services/postService');
+
+// 게시글 등록
+const postInput = async (req, res) => {
```

Comment:
함수 이름을 지을 때 행위+대상으로 지어주시면 좋습니다.
```suggestion
const createPost = async (req, res) => {
```

Reply:
수정했습니다.